src/main/resources/i18n/aerospace/common.properties

```diff
@@ -267,6 +267,11 @@ APPROVED=Approved
 DECLINED=Denied
 # parasoft-end-suppress PROPS.DUPV
 REQUESTED_BY=Requested by
+NON_REPEATABLE_OPERATIONS=You have already modified this order status and cannot operate it again
```

Comment:
Add . at the end of the sentences to keep consistent with other toast sentences.